PillarChap/Pillar.pillar

```diff
@@ -13,18 +17,19 @@ screenshot below (see Figure *@voyageDocExample*) shows a part of the
 +An example Pillar output>file://figures/voyageDocExample-small.png|label=voyageDocExample|width=30+
 
 Pillar has many features:
-- simple markup syntax with references, tables, pictures, captions...);
-- export to HTML, LaTeX, Markdown and Pillar itself;
-- customization of the export through a dedicated *STON>http://smalltalkhub.com/#!/~SvenVanCaekenberghe/STON* configuration file;
-- support of templates using the *Mustache>http://smalltalkhub.com/#!/~NorbertHartl/Mustache* templating engine;
-- syntax-highlighting of generated code blocks (not yet in LaTeX);
-- configurable numbering of section titles and figures;
+- simple markup syntax with references, tables, pictures, captions...)
```

Comment:
It seems weird to have semi-colons inside a list no ? The semi-colon is a way to separate two elements, but inside a list it is already separate.
Or is it an English convention ?